devtools/conda-envs/test_env.yaml

```diff
@@ -49,3 +47,4 @@ dependencies:
     - types-toml
     - types-PyYAML
     - mongo-types
+    - git+https://github.com/openforcefield/openff-units.git@element-module
```

Comment:
This would be in a release if this PR is approved; I just don't want to make a release for an idea that might not see the light of day.

Reply:
Thanks - did you want to review https://github.com/openforcefield/openff-units/pull/14, or should I merge + release? Given this PR is in good shape, I don't think it's required.

Reply:
merged & released!